### PR TITLE
Oddshot.tv HTML5 Player

### PR DESCRIPTION
**Pull request by nicememe**
- Replaced the `<video>` tag with `<iframe>` including the new embedded player by Oddshot.tv.
- Gulp code style compliance met.

### CODE REVIEW

File: lib/modules/hosts/oddshot.js [hunk 1]
```diff
@@ -1,50 +1,26 @@
...
-		var sources = [{
-			'file': 'https://d301dinc95ec5f.cloudfront.net/capture/' + info + '.shot.mp4',
-			'type': 'video/mp4'
-		}];
-
-		$(elem).data('sources', sources);
-		if (RESUtils.pageType() === 'linklist') {
-			$(elem).closest('.thing').find('.thumbnail').attr('href', elem.href);
+		if (modules['showImages'].options.autoplayVideo.value) {
+			if ($(elem).closest('.md').find('.expando-button.video').length === 0) info += '?autoplay=true';
```
Comment (andytuba) [inline lib/modules/hosts/oddshot.js]: What's the intent of this check? Should autoplay only happen when there's one video in a comment?

Comment (nicememe) [inline lib/modules/hosts/oddshot.js]: @andytuba The check is there so the video does not auto play if there's another expando is opened.

Comment (erikdesjardins) [inline lib/modules/hosts/oddshot.js]: Yeah, it's done for most video expandos, e.g. [youtube](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/9ed129d0e322ba00214f9c379f12682eb2dfcba1/lib/modules/hosts/youtube.js#L53), [vimeo](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/9ed129d0e322ba00214f9c379f12682eb2dfcba1/lib/modules/hosts/vimeo.js#L27)

Comment (andytuba) [inline lib/modules/hosts/oddshot.js]: Cool, thanks.

**Comment by andytuba** [comment]
Merged to release/4.6.1

I'll comment again on this PR when it's released.

**Comment by andytuba** [comment]
v4.6.1 rolling out now!

**Pull request closed without merging**